spiffworkflow-frontend/test/browser/tasks/test_can_complete_and_navigate_a_form.py

```
@@ -0,0 +1,117 @@
+import re
+from playwright.sync_api import expect, Page, BrowserContext
+
+from helpers.login import login, logout, BASE_URL
+from helpers.playwright_setup import browser_context  # fixture
```

Ruff (0.11.9)

`helpers.playwright_setup.browser_context` imported but unused

Remove unused import: `helpers.playwright_setup.browser_context`

(F401)